guides/keras_nlp/quick_tour.py

```diff
@@ -0,0 +1,610 @@
+"""
+Title: KerasNLP quick tour
+Author: [jbischof](https://github.com/jbischof)
+Date created: 2022-12-15
+Last modified: 2022-12-15
+Description: An introduction to the KerasNLP API.
+"""
+"""
+<a
+href="https://colab.research.google.com/github/jbischof/keras-io/blob/quickstart/guides/ke
+ras_nlp/keras_nlp_quick_tour.ipynb" target="_parent"><img
+src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+"""
+
+import keras_nlp
+import tensorflow as tf
+from tensorflow import keras
+import tensorflow_datasets as tfds
+
+# Use mixed precision for optimal performance
+keras.mixed_precision.set_global_policy("mixed_float16")
+
+"""
+# KerasNLP: Modular NLP Workflows for Keras
+
+
+`keras-nlp` is a natural language processing library that supports users through their
+entire development cycle. Our workflows are built from modular components that have SoTA
+preset weights and architectures when used out-of-the-box and are easily customizable
+when more control is needed.
+
+This library is an extension of the core `keras` API; all high level modules are `Layers`
+or `Models`. If you are familiar with `keras`, congratulations! You already understand
+most of `keras-nlp`.
+
+This guide demonstrates our modular approach using a sentiment analysis example at six
+levels of complexity:
+* Inference with a pretrained classifier
+* Fine tuning a pretrained backbone
+* Fine tuning with user-controlled preprocessing
+* Fine tuning a custom model
+* Pretraining a backbone model
+* Build and train your own transformer from scratch
+
+Throughout our guide we use Professor Keras, the official Keras mascot, as a visual
+reference for the complexity of the material:
+
+![picture](https://drive.google.com/uc?id=1d14Qpmfgjf6zu4z30HBaonH8PYDHgVoU)
+
+
+"""
+
+"""
+# API quickstart
```

Comment:
Every heading from here and below should be one level lower. E.g. `#` -> `##`, and `##` -> `###`.

Reply:
I made these changes locally before pushing the preview. So that is what it looks like after the change. Before looked pretty wonky.